config/intro-template.md.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+const upperFirst = require('lodash.upperfirst')
+const camelCase = require('lodash.camelcase')
+
+function exampleTmpl (example) {
+  if (!example) {
+    return ''
+  }
+
+  return `
+
+## Quick Example
+
+\`\`\`js
+${example}
+\`\`\`
+
+There are more available, so take a look at the docs below for a full list. This documentation aims to be comprehensive, so if you feel anything is missing please create a GitHub issue for it.`
+}
+
+module.exports = (name, url, example) => `
+Installable via  \`npm install --save ${name}\`, it can also be used directly in the browser.
+
+${exampleTmpl(example)}
+
+## Download
+
+The source is available for download from [GitHub](${url}). Alternatively, you can install using npm:
+
+\`\`\`bash
+$ npm install --save ${name}
+\`\`\`
+
+You can then \`require()\` ${name} as normal:
+
+\`\`\`js
+const ${camelCase(name.replace(/-/g, ' '))} = require('${name}')
```

Comment:
Nevermind me, camelCase fixes all this, we don't even need the `.replace` call...